opm/material/thermal/EclSpecrockLawParams.hpp

```diff
@@ -62,7 +62,7 @@ class EclSpecrockLawParams : public EnsureFinalized
         // integrate the heat capacity to compute the internal energy
         Scalar curU = temperature[0]*heatCapacity[0];
         std::vector<Scalar> T(temperature.size());
-        std::vector<Scalar> u(heatCapacity.size());
+        std::vector<Scalar> u(temperature.size());
```

Comment:
I don't see the point of this change, given the assert on line 60?

Reply:
the point is to make it a bit clearer that this is a new array, probably

```
assert(heatCapacity.size() == temperature.size());
unsigned n = temperature.size();
std::vector<Scalar> T(n);
std::vector<Scalar> u(n);
```

would be even better.